frontend-resources/pom.xml

```diff
@@ -80,6 +80,16 @@
 							<arguments>install --frozen-lockfile --non-interactive --ignore-engines</arguments>
 						</configuration>
 					</execution>
+					<execution>
+						<id>copy .env</id>
+						<goals>
+							<goal>cp</goal>
```

Comment:
Well, somehow this won't work :-( Quite annoying that copying a single file with maven is so tricky.

Means that this PR is probably on hold, until we manage to make frontend pipelines work without maven.